theseus_gui/src/helpers/utils.js

```diff
@@ -0,0 +1,12 @@
+export const releaseColor = (releaseType) => {
+  switch (releaseType) {
+    case "release":
+      return "green";
+    case "beta":
+      return "orange";
+    case "alpha":
+      return "red";
+    default:
+      return "";
+  }
```

Comment:
feels weird to have this as a func, also iirc the badge auto sets the color when the names are release, beta, and alpha, so this code isn't needed